plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobContainer.java

```diff
@@ -117,6 +126,30 @@ public void deleteBlob(String blobName) throws IOException {
         }
     }
 
+    @Override
+    public void deleteBlobsIgnoringIfNotExists(List<String> blobNames) throws IOException {
```

Comment:
Can we add the comment that Azure does not support batch API, that's why we have no other choice than using parallel execution?